index.html

```diff
@@ -0,0 +1,274 @@
+                  src="./new-img/exhibition.svg"
+                  alt="exhibition"
+                />
+              </div>
+              <h3>Resilien</h3>
+              <p>
+                Resilience and Hope: Strength in Adversity a. Inspiring
+                photographs and stories showcasing the resilience and
+                determination of asylum seekers to rebuild their lives despite
+                the challenges they face.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/forum.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Debate</h3>
+              <p>
+                Cultural Exchange and Understanding: Participants engage in
+                activities promoting intercultural dialogue, breaking down
+                barriers, and fostering understanding between asylum seekers and
+                the host community.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/station.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Seminar</h3>
+              <p>
+                Each group will go through a series of simulated experiences,
+                such as navigating a complex bureaucratic process, facing
+                language barriers, or living in crowded conditions.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/ignite.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Kindled</h3>
+              <p>
+                attending ALD events offers the opportunity to network with ALD
+                affiliates from around the world, fostering ongoing connections
+                and potential collaborations beyond the duration of the summit.
+              </p>
+            </div>
+          </div>
+          <div class="see-program">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+          </div>
+          <div class="join-btn">
+            <button type="button" class="join btn btn-lg">Join as Now!</button>
+          </div>
+        </section>
+      </section>
+    </article>
+    <!-- Main Program Ends -->
+    <!-- Speakers Strats -->
+    <article id="featured-speakers" class="container featured-speakers">
+      <section class="speaker-intro">
+        <h2>Featured Speakers</h2>
+        <div class="speaker-intro-border"></div>
+      </section>
+      <div id="speaker-wrapper" class="row speaker-wrapper"></div>
```

Comment:
- [ ] Great work reducing the featured speakers to **_two (2) speakers_** for the mobile version. However, you forgot to include the _**More**_ button for the mobile version. You don't have to implement the functionality of the _**More**_ button **_(if you can, it will be great for the project and improve your skill)_**, **_but you should include it as a placeholder button_**. Kindly include the _**More**_ button and design it to be identical to that of the design template provided. 👍 😎 

### Your Design

![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/77106962/8febc5c4-9b1f-401a-8edc-9d671950b673)

### Behance Template

![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/77106962/e6248ab1-a011-45ec-8e59-c7d5d2f228f7)